src/bindings.cpp

```diff
@@ -23,7 +23,7 @@ namespace madrona_gpudrive
         m.attr("kMaxAgentCount") = consts::kMaxAgentCount;
         m.attr("kMaxRoadEntityCount") = consts::kMaxRoadEntityCount;
         m.attr("kMaxAgentMapObservationsCount") = consts::kMaxAgentMapObservationsCount;
-        m.attr("episodeLen") = consts::episodeLen;
+        m.attr("maxEpisodeLen") = consts::maxEpisodeLength;
```

Comment:
Can't we stick to the original naming of episodeLen? The fact that an agent can be terminated before the episodeLen seems implied